rover-apps/common/src/main.cpp

```diff
@@ -0,0 +1,90 @@
+#include "AppConfig.h"
+#include "Logger.h"
+#include "mbed.h"
+
+Thread periodic_10s_thread(osPriorityNormal1);
+Thread periodic_1s_thread(osPriorityNormal2);
+Thread periodic_100ms_thread(osPriorityNormal3);
+Thread periodic_10ms_thread(osPriorityNormal4);
+Thread periodic_1ms_thread(osPriorityNormal5);
```

Comment:
I think these should be a lot higher priority. Since almost all threads should be replaced with these periodic functions, its fairly important that these are high priority. The majority of these threads should also be sleeping most of the time too.

```suggestion
Thread periodic_10s_thread(osPriorityHigh);
Thread periodic_1s_thread(osPriorityHigh2);
Thread periodic_100ms_thread(osPriorityHigh4);
Thread periodic_10ms_thread(osPriorityHigh6);
Thread periodic_1ms_thread(osPriorityRealtime);
```